assets/javascript/app.js

```diff
@@ -74,127 +74,61 @@ var page = {
 
 			var preferences = exclusion.preferences;
 			
```

Comment:
None of the below code is different other than it now being part of a 'done' promise function.